.github/workflows/build.yml

```diff
@@ -52,29 +52,32 @@ jobs:
       - name: Check CS (Code Style)
         run: composer lint
 
-  test-momento:
+  test:
     runs-on: ubuntu-latest
+    services:
+      redis:
+        image: redis:latest
+        ports:
+          - 6379:6379
+
     strategy:
       matrix:
+        backend: [momento, redis]
```

Comment:
Instead of a matrix, you can just have two steps one after the other: one that tests redis and one that tests momento. Then there's no need to parameterize the job.